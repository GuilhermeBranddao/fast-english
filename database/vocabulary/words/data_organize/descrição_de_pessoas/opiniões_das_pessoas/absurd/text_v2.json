{
  "pergunta_pt-br": "absurdo",
  "uso_da_linguagem": "Adjetivo",
  "tradução_en": "absurd",
  "fonetica": "/ab'saird/"
}