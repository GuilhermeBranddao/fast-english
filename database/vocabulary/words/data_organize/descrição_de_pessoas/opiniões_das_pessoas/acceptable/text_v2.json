{
  "pergunta_pt-br": "aceitável",
  "uso_da_linguagem": "A Adjetivo aoe",
  "tradução_en": "acceptable",
  "fonetica": "/akseptabl/"
}