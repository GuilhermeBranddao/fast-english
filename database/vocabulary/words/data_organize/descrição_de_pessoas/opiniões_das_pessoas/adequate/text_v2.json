{
  "pergunta_pt-br": "adequado",
  "uso_da_linguagem": "A Adjetivo ...",
  "tradução_en": "adequate",
  "fonetica": "/zedikwat/"
}