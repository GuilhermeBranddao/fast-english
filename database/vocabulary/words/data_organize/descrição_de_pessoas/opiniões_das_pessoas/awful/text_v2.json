{
  "pergunta_pt-br": "horrível",
  "uso_da_linguagem": "A Adjetivo ...",
  "tradução_en": "awful",
  "fonetica": "fofi/"
}