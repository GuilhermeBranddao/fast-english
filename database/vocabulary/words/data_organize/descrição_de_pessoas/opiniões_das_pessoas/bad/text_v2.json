{
  "pergunta_pt-br": "mau",
  "uso_da_linguagem": "Adjetivo",
  "tradução_en": "bad",
  "fonetica": "/beed/"
}