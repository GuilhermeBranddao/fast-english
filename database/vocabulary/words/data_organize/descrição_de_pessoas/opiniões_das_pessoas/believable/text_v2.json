{
  "pergunta_pt-br": "credível",
  "uso_da_linguagem": "A Adjetivo aoe",
  "tradução_en": "believable",
  "fonetica": "/brili:vabl/"
}