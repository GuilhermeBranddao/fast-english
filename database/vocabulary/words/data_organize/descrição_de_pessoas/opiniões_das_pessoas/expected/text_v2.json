{
  "pergunta_pt-br": "expectável",
  "uso_da_linguagem": "A Adjetivo aoe",
  "tradução_en": "expected",
  "fonetica": "/ikspektid/"
}